docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -162,12 +162,14 @@ Test launcher script `7zip/tests/test.sh`:
 
 ```bash
 #!/bin/sh
+set -euo pipefail
+
 #/ Usage: test.sh <pkg_ident>
 #/
 #/ Example: test.sh core/7zip/1.2.3/20181108151533
 #/
 
-set -euo pipefail
+TESTDIR="$(dirname "${0}")"
```

Comment:
Declaring this variable might suggest it's used somewhere else, and it's not. The previous version (that did not declare a variable) is maybe more expressive that we're just getting a single-use out of the dirname

Reply:
Fixed.

```diff
--- a/docs/dev/policy_documents/standards-for-tests.md
+++ b/docs/dev/policy_documents/standards-for-tests.md
@@ -172,8 +172,6 @@ set -euo pipefail
 #/ Example: test.sh core/7zip/1.2.3/20181108151533
 #/

-TESTDIR="$(dirname "${0}")"
-
 if [[ -z "${1:-}" ]]; then
   grep '^#/' < "${0}" | cut -c4-
   exit 1
@@ -184,7 +182,8 @@ export TEST_PKG_IDENT

 hab pkg install core/bats --binlink
 hab pkg install "${TEST_PKG_IDENT}"
-bats "${TESTDIR}/test.bats"
+
+bats "$(dirname "${0}")"/test.bats
 ```